src/test/java/domain/CarTest.java

```diff
@@ -0,0 +1,35 @@
+package domain;
+
+import static org.assertj.core.api.Assertions.*;
+
+import org.junit.jupiter.api.Test;
+
+class CarTest {
+  private Car car1=new Car("car1");
+
+  @Test
+  void 자동차_생성() {
+    assertThat(car1).isNotNull();
+  }
+
+  @Test
+  void 자동차_이동() {
+    car1.move(7);
+    int currentPosition = car1.getPosition();
+    assertThat(currentPosition).isEqualTo(1);
+  }
+
+  @Test
+  void 자동차_멈춤() {
+    car1.move(3);
+    int currentPosition = car1.getPosition();
+    assertThat(currentPosition).isEqualTo(0);
+  }
+  @Test
+  void 자동차_위치_비교(){
+    Car car2=new Car("car2");
```

Comment:
```suggestion
    Car car2 = new Car("car2");
```
맥북이시라면 cmd + option +  l 을 통해서 컨벤션 정리가 되는데 
한번씩 해보면 좋을것 같습니다
구글링해보면 intellij 기본 설정으로도 할 수 있을거에요